cartographer_ros/src/ros_log_sink.cpp

```diff
@@ -33,6 +33,11 @@ const char* GetBasename(const char* filepath) {
   return base ? (base + 1) : filepath;
 }
 
+std::chrono::system_clock::time_point ConvertTmToTimePoint(const std::tm& tm) {
+    std::time_t timeT = std::mktime(const_cast<std::tm*>(&tm)); // Convert std::tm to time_t
+    return std::chrono::system_clock::from_time_t(timeT);      // Convert time_t to time_point
+}
```

Comment:
CI is not happy with this in the "default" case, since it is unused.  Maybe wrap this in the `#if defined(ROS_CARTOGRAPHER_GLOG_GET_070)` macro?

Reply:
Done in https://github.com/ros2/cartographer_ros/pull/76/commits/eaeffcfec704211136b52b9ec63e2d91e58216e6 .